src/Symfony/Component/HttpClient/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.4
+---
+
+ * Add option `crypto_method` to set the minimum SSL version and make it default to TLSv1.2
```

Comment:
Defaulting to 1.2 is a breaking change. I think we need to default to 1.0, then change this to 1.2 in v7.

Reply:
Unless we treat this as a security/bug fix... which means we need to target an earlier branch. ;)

Reply:
I think we should consider this as an improvement, that's why I target 6.4 (6.3 is in feature freeze mode)